src/markdown-pages/build-apps/build-visualization.mdx

```diff
@@ -0,0 +1,210 @@
+---
+path: '/build-apps/build-visualization'
+duration: '15 min'
+title: 'Build a custom visualization for dashboards'
+template: 'GuideTemplate'
+description: 'Create custom data views'
+resources:
+  - title: New Relic One VSCode extension
+    url: https://marketplace.visualstudio.com/items?itemName=new-relic.nr1
+  - title: 'Build on New Relic One'
+    url: https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/build-new-relic-one/new-relic-one-build-your-own-custom-new-relic-one-application
+  - title: New Relic VSCode extension pack
+    url: https://marketplace.visualstudio.com/items?itemName=new-relic.new-relic-extension-pack
+tags:
+  - nr1 cli
+  - NR One Catalog
+  - Subscribe visualizations
+---
+
+<Intro>
+
+Building your own visualizations for New Relic dashboards enables you and your organization to query data from any source and display it on your dashboard. In this guide, you'll learn how to use the CLI to generate a visualization file structure, run the visualization locally in New Relic One's Visualization Explorer where you can quickly test and iterate, and deploy the visualization for you and others to use on dashboards.
+
+</Intro>
+
+## Before you begin
+
+To get started, make sure you have a [New Relic](https://newrelic.com/signup/) account.
+
+To develop projects, you need the New Relic One CLI (command line interface). If you haven't already installed it, do the following:
+
+- Install [Node.js](https://nodejs.org/en/download/).
+- Complete all the steps in the [CLI quick start](https://one.newrelic.com/launcher/developer-center.launcher?pane=eyJuZXJkbGV0SWQiOiJkZXZlbG9wZXItY2VudGVyLmRldmVsb3Blci1jZW50ZXIifQ==).
+
+## Create a new visualization
+
+In this first set of steps you use the CLI to quickly generate the boilerplate for a new visualization.
+
+<Steps>
+
+  <Step>
+
+Ensure you're working with the latest version of the New Relic One CLI:
```

Comment:
This may be better surfaced as a call out, by using our callout component: https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/README.md#callout